spec/acceptance/validate_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper_acceptance'
+require 'tmpdir'
+require 'fileutils'
+
+describe 'Running validation' do
+  context 'outside of a module' do
```

Comment:
Once we have a uniform behaviour above, you can extract this (and https://github.com/puppetlabs/pdk/blob/master/spec/acceptance/new_class_spec.rb#L92) into a shared examples, so this becomes `it_behaves_like 'requires running in a module directory'`.